DEVELOPER_STARTING_GUIDE.md

```diff
@@ -0,0 +1,185 @@
+        - `accoridion.tests.ts`: Suite of jest tests for the accordion component. Critical for deployment and maintenance.
+        - `accordion.stories.ts`: Collection of Storybook stories that are primarily used for visual testing with Chromatic. Any **samples** are added as stories here.
+      - `docs`: all ".mdx" doc files that appear in the "DOCS" section of our Storybook
+        - `migration`: Individual component migration guides
+      - `internal`: A set of utilities that are reusable internally. Check here should you need something like a debounce, it may have already been implemented.
+      - **`patterns`**: Demonstrate how to combine several components to solve a specific problem. Added as stories in the "PATTERNS" section.
+      - **`styles`**: These are standalone CSS files. They don't provide any logic and are often non-interactive. Added as stories in the "STYLES" section.
+
+## Technologies
+
+- pnpm
+- Web Components
+- Lit JS
+- Tailwind
+- Storybook
+- Jest
+- Chromatic
+- Vite
+- PostCSS
+
+//TBD: Do we keep them and apply them even thought the nome does not come from the SOLID principles?
+
+## SOLID Design Principles in Software Development
+
+[SOLID](https://www.freecodecamp.org/news/solid-design-principles-in-software-development/) is a set of five design principles that aid in designing robust, testable, extensible, and maintainable object-oriented software systems. Each principle addresses specific challenges in software development.
+
+- **Single Responsibility Principle (SRP)**
+
+  - A class, module, or function should have only one reason to change, i.e., it should perform a single responsibility.
+  - Example: Separating classes for animal information, sound, and feeding.
+
+- **Open-Closed Principle (OCP)**
+
+  - Classes, modules, and functions should be open for extension but closed for modification.
+  - Example: Using abstraction to add new animal types without modifying existing code.
+
+- **Liskov Substitution Principle (LSP)**
+
+  - Child classes must be substitutable for their parent classes, ensuring seamless replacement.
+  - Example: Dog and Cat classes replacing the parent Animal class.
+
+- **Interface Segregation Principle (ISP)**
+
+  - Clients should not be forced to implement interfaces or methods they do not use.
+  - Example: Breaking down large interfaces into smaller, specific ones.
+
+- **Dependency Inversion Principle (DIP)**
+  - High-level modules should not depend on low-level modules; both should depend on abstractions.
+  - Example: Using abstractions to decouple high-level and low-level modules.
+
+Implementing SOLID principles leads to a large but maintainable codebase, enabling developers to make changes without causing major issues. These principles contribute to bug-free, flexible, scalable, and reusable software systems.
```

Comment:
```suggestion
## SOLID Design Principles in Software Development

After defining our own principles and decided for the name Solid Design System, we found the [SOLID Design Principles](https://www.freecodecamp.org/news/solid-design-principles-in-software-development/). A set of five design principles that aid in designing robust, testable, extensible, and maintainable object-oriented software systems. Each principle addresses specific challenges in software development.

These principles are not developed by us and the connection is more 'unintentional'. Nevertheless, the strong and deep connection between our principles and the ones mentioned above is definitely worth sharing here and we highly recommend to get familiar with them as well.
```

Reply:
```suggestion
## SOLID Design Principles in Software Development

After defining our own principles and decided for the name Solid Design System, we realized the close connection to [SOLID Design Principles](https://www.freecodecamp.org/news/solid-design-principles-in-software-development/), a set of five design principles that aid in designing robust, testable, extensible, and maintainable object-oriented software systems. Each principle addresses specific challenges in software development.
These principles are not developed by us and the connection is more 'unintentional'. Nevertheless, the strong and deep connection between our principles and the ones mentioned above is definitely worth sharing here and we highly recommend to get familiar with them as well.
```

I just a little bit the wording – so that it doesn't sound like we never heard of it.  :)